content/guides/ecommerce-connection/step0-who-is-your-connector-for.md

```diff
@@ -35,9 +35,9 @@ It depends on your business model. If you require your client to install the con
 
 * A **full documentation** about **the limits of the connector** in terms of **scalability** and **performance**.
 
-* A **easy to follow configuration process** to set up the connection between Akeneo PIM and eCommerce solution as well as parameters to configure the connector. It requires a documentation that explains the methodology on how to configure your connector compared to business needs.
+* An **easy to follow configuration process** to set up the connection between Akeneo PIM and eCommerce solution as well as parameters to configure the connector. It requires a documentation that explains the methodology on how to configure your connector compared to business needs.
 
-* An **friendly import process** to perform the first import of PIM data into the eCommerce solution. Due to the volume of data to be imported, this may require special support from you.
+* A **friendly import process** to perform the first import of PIM data into the eCommerce solution. Due to the volume of data to be imported, this may require special support from you.
```

Comment:
```suggestion
* A **friendly import process** to perform the first import of the PIM data into your eCommerce solution. Due to the volume of data to be imported, this may require special support from your end.
```